.github/labels.yml

```
@@ -0,0 +1,97 @@
+---
+# Rather than breaking up descriptions into multiline strings we disable that
+# specific rule in yamllint for this file.
+# yamllint disable rule:line-length
+- color: f15a53
+  description: Pull requests that update Ansible code
+  name: ansible
+- color: eb6420
+  description: This issue or pull request is awaiting the outcome of another issue or pull request
+  name: blocked
+- color: "000000"
+  description: This issue or pull request involves changes to existing functionality
+  name: breaking change
+- color: d73a4a
+  description: This issue or pull request addresses broken functionality
+  name: bug
+- color: 07648d
+  description: This issue will be advertised on code.gov's Open Tasks page (https://code.gov/open-tasks)
+  name: code.gov
+- color: ffd343
+  description: Pull requests that update cyhy-core command-line tools
```

An alternative if we want one description to cover both cyhy-core and cyhy-reports:
```suggestion
  description: Pull requests that update CyHy command-line tools
```

---

I think this is the better approach for conformity actually.